packages/expo-build-properties/src/android.ts

```diff
@@ -1,4 +1,10 @@
-import { AndroidConfig, ConfigPlugin, History, withDangerousMod } from 'expo/config-plugins';
+import {
+  AndroidConfig,
+  ConfigPlugin,
+  History,
+  withDangerousMod,
+  withGradleProperties,
+} from '@expo/config-plugins';
```

Comment:
Do we need to change this import from `'expo/config-plugins'` to `@expo/config-plugins`?

Reply:
Yes. We should be using @expo/config-plugins everywhere. I'll include these changes in the fixup